rwgame/debugstate.cpp

```diff
@@ -80,7 +80,7 @@ DebugState::DebugState(RWGame* game, const glm::vec3& vp, const glm::quat& vd)
 		auto spawnPos = game->getPlayer()->getCharacter()->getPosition();
 		spawnPos += playerRot * glm::vec3(0.f, 3.f, 0.f);
 		auto spawnRot = glm::quat(glm::vec3(0.f, 0.f, glm::roll(playerRot) + glm::half_pi<float>()));
-		auto car = game->getWorld()->createVehicle(136, spawnPos, spawnRot);
+		auto car = game->getWorld()->createVehicle(110, spawnPos, spawnRot);
```

Comment:
Nope, should be handled differently? Maybe just create a second debug function for now? (Banshee, Police Car, Taxi, Firetruck, Ambulance)